### DISCUSSION TITLE
Topaz wrapper referring back to .eer movies when inputting motion corrected micrographs

### DISCUSSION THREAD

**Discussion by ArmandoPach** [tbepler/topaz#147] [cryoSPARC]
Hi!

I am trying to train a new topaz denoising model for my micrographs (the provided model did not give me the expected results, blurring my particles). Since .eer files are not supported yet, I input the motion-corrected micrographs, but I get a .eer error.

The problem is that I guess CS refers back to the .eer files, and the job fails. Are there any recommendations on how to circumvent this?

Best.
Armando

**Comment by tbepler** [comment]
Could you share an example of the file that's causing the error? Have you converted your EER micrographs into regular MRC files?
